dev/generator/main.go

```diff
@@ -277,6 +277,13 @@ func encodedSavedObject(data []byte) (string, error) {
 			continue
 		}
 
+		// It may happen that some objects existing in example directory might be already encoded.
+		// In this case skip the encoding.
+		_, isString := out.(string)
+		if isString {
+			continue
```

Comment:
As I would consider this an invalid package it would be nice if we could also add a log output here so we can fix the packages. Or more radical, return an error?

Reply:
Fixed.